src/main/java/nl/knaw/dans/managedeposit/core/DepositProperties.java

```diff
@@ -172,4 +173,20 @@ public boolean equals(Object o) {
     public int hashCode() {
         return 31 * depositId.hashCode() + depositor.hashCode();
     }
+
+    @Override
+    public String toString() {
+        return "DepositProperties{" +
+            "depositor='" + depositor + '\'' +
```

Comment:
```suggestion
            "depositor='" + depositor + "'" +
```
looks clearer to me than escaping.

Reply:
okay. I'm committing your suggestion. Thanks.